scripts/install-results.sh

```diff
@@ -27,19 +10,8 @@ metadata:
   name: result
 spec:
   targetNamespace: openshift-pipelines
-  logging_pvc_name: tekton-logs
-  logs_api: true
-  log_level: debug
-  db_port: 5432
-  db_host: tekton-results-postgres-service.openshift-pipelines.svc.cluster.local
-  logs_path: /logs
-  logs_type: File
-  logs_buffer_size: 32768
-  auth_disable: true
-  tls_hostname_override: tekton-results-api-service.openshift-pipelines.svc.cluster.local
-  db_enable_auto_migration: true
-  server_port: 8080
-  prometheus_port: 9090
+  loki_stack_name: logging-loki
+  loki_stack_namespace: openshift-logging
```

Comment:
Can you please add also under spec ?

`auth_disable: true`

Reply:
added.